lib/elixir/pages/Naming Conventions.md

```diff
@@ -0,0 +1,90 @@
+# Naming Conventions
+
+## Naming modules
+
+Elixir module names are written in "upper camel case", like `OptionParser`.
+
+Capital letters are kept in abbreviations, like `ExUnit.CaptureIO` or `Mix.SCM`.
```

Comment:
It was was a confusing gotcha when we initially tried `MyApp.CLI`.  Do you think Elixir could be tolerant of the alternate capitalization?